plugins/notifications-common/src/types.ts

```diff
@@ -33,7 +33,7 @@ export type NotificationPayload = {
 /** @public */
 export type Notification = {
   id: string;
-  user: string;
+  user: string | null;
```

Comment:
Is the `null` for broadcast?
Can there be another non-broadcast flow where the `user` is `null` as well?
We should document that.

Reply:
Yes, it's for the broadcast. It's only null for broadcast notifications. We could use empty string as this far, but I think null makes more sense.

Reply:
Can you update the documentation for that?

Reply:
Okay